docs/guides/publish_mint_packages.md

```diff
@@ -0,0 +1,76 @@
+Once you have finished developing and testing your service locally, is time to publish the developed software packages on a remote registry (so that they become publicly available) and mint them in the [Autonolas Protocol](https://docs.autonolas.network/protocol/).
+
+<figure markdown>
+![](../images/development_process_mint_packages_on_chain.svg)
+<figcaption>Part of the development process covered in this guide</figcaption>
+</figure>
+
+The [Autonolas Protocol](https://docs.autonolas.network/protocol/) provides primitives to create, operate and secure agent services on-chain. It also provides a mechanism that incentivizes their creation and rewards developers and service operators proportionally for their efforts to support the growth of the Autonolas ecosystem. The protocol comprises the **on-chain registry**, a collection of ERC721 smart contracts that manage the registration of software packages (components, agents and services). The registry stores a representation of the packages minted in the form of NFTs.
+Minting packages on-chain is a requirement to secure and use them in the [Autonolas Protocol](https://docs.autonolas.network/protocol/).
+
+You can mint packages using the [Autonolas Protocol web app](https://protocol.autonolas.network/) or the Open Autonomy CLI.
+
+## What will you learn
+
+This guide covers step 6 of the [development process](./overview_of_the_development_process.md). You will learn how to publish the software packages developed in the local registry (components, agents and services) to the remote registry, and how to mint them on-chain.
+
+You must ensure that your machine satisfies the framework requirements and that you have [set up the framework](./set_up.md#set-up-the-framework) and [a local registry](./set_up.md#set-up-the-local-registry). As a result you should have a Pipenv workspace folder with a local registry (`./packages`) in it.
+
+## Publish packages to the remote registry
+
+If you have developed your components in the local registry, then the index file (`./packages/packages.json`) should contain a reference for them in the `dev` section, for example:
+
+```json
+{
+    "dev": {
+        "service/your_name/your_service/0.1.0": "bafybei0000000000000000000000000000000000000000000000000000",
+        "agent/your_name/your_agent/0.1.0": "bafybei0000000000000000000000000000000000000000000000000000",
+        "skill/your_name/your_fsm_app/0.1.0": "bafybei0000000000000000000000000000000000000000000000000000"
+    },
+    "third_party": {
+        (...)
+    }
+}
+```
+
+The easiest way to have your components published in the remote registry is by executing the following commands:
+
+```bash
```

Comment:
IMO this would be better as an itemized list. The problem with the current formatting is that it invites the developer to run these commands straight away, which I think is not what you mean.

Reply:
rewritten